src/library/playlistfeature.cpp

```diff
@@ -173,8 +173,7 @@ void PlaylistFeature::buildPlaylistList() {
             playlistTableModel.index(row, durationColumn)).toInt();
         m_playlistList.append(qMakePair(id, QString("%1 (%2) %3")
                                         .arg(name, QString::number(count),
-                                                mixxx::Duration::formatSeconds(duration))));
-    }
```

Comment:
is this curly brace lost?

Reply:
I don't know how it ended up there, but it's the closing brace of line 165. :)

Reply:
Correction: You are right, when I adjusted the formatting of my addition I removed a curly brace! I had just compiled to test and before committing made one last tweak (also known as a bug). just fixed and squashed.